### PR TITLE
newsletter june 2024

### PR DESCRIPTION
**Pull request by ubdbra001**

### CODE REVIEW

**Comment by twinkarma** [comment]
I'd forgotten to put it in a newswire again but can you help promote best practies in ai event this month too please! https://rse.shef.ac.uk/events/seminar-2024-07-05-best-practices-in-ai-afternoon.html

File: _newsletters/2024-06-05-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+Welcome to the June 2024 newsletter for the research software community at The University of Sheffield, featuring
+news, opportunities, events and training for you.
+
+
+### News
+
+- Would you like 6 months of free RSE time on your project? The [2024 RSE collaboration call for proposals](https://rse.shef.ac.uk/collaboration/RSEtime_call2024/) is open. __Deadline: 28th June__
+
+- [Responsible AI UK: skills programme](https://www.ukri.org/opportunity/responsible-ai-uk-skills-programme/?utm_medium=email&utm_source=govdelivery): Funding to develop open educational resources for the training of artificial intelligence (AI) developers, users, managers, the public and other stakeholders. Proposals may include research into the creation of resources, responding to responsible AI skills critical challenges, including issues related to upskilling, reskilling, pedagogy, or educational delivery. __Deadline: 14 June 2024__
```
Comment (twinkarma) [inline _newsletters/2024-06-05-newsletter.md]: Should the tracking params be removed from the url? `?utm_medium=email&utm_source=govdelivery`

**Comment by ubdbra001** [comment]
Will do, shall I keep the call for speakers in too?

**Review by twinkarma — APPROVED** [review]
Looks good to me! Just a very minor comment on one of the links

**Comment by twinkarma** [comment]
> Will do, shall I keep the call for speakers in too?

@ubdbra001 Yes please we're still looking for more speakers

**Comment by twinkarma** [comment]
I feel that we should remove the lunchbytes sections as we haven't held one for at least a year or two. Do we ever get anyone submitting a talk for it?

File: _newsletters/2024-06-05-newsletter.md (hunk 1 above)
Comment (ubdbra001) [reply _newsletters/2024-06-05-newsletter.md]: Probably a good idea

**Comment by ubdbra001** [comment]
> I feel that we should remove the lunchbytes sections as we haven't held one for at least a year or two. Do we ever get anyone submitting a talk for it?

Agree, but maybe something to bring up for the next team meeting? With a view to removing it in the next newsletter?

**Pull request merged**